.ci/scripts/dart_unit_tests_win32.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Exclude flutter_image because its tests need a test server, so are run via custom_package_tests.
+# Exclude xgd_directories because it is a Linux only package.
```

Comment:
Please use an exclusion file in /script/config/; it's much easier to maintain and comment that way.